docs/DeveloperGuide.md

```diff
@@ -403,7 +403,22 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
    Use case ends.
 
-### UC08: Edit a book on LibTask
+### UC08: Find books on LibTask
+
+**MSS**
+1. User requests to find books which matches a query. 
+
+2. LibTask shows the books that match the query
+
+    Use case ends
+
+**Extension**
+
+* 1a The given query is invalid
+  * 1a1 LibTask shows an error message
+    Use case resumes from step 1
+
```

Comment:
I think another extension could be that no books fulfill the find criteria. Then LibTask shows an empty list. Take for example from the use case 04: Find a patron on LibTask.